rise-storage.html

```diff
@@ -1330,8 +1330,7 @@
         decodedUrl = url.substring(0, end !== -1 ? end : url.length);
       }
 
-      // Decode again to handle double slash encodings
```

Comment:
@rodrigopavezi If I remember correctly, this was needed because the file names contained in Storage could have spaces or other characters which require encoding. This means the GCS url would contain this file name parameter encoded in base64. Considering the full GCS url is encoded, to get the "clean" file name back you need to decode it again (otherwise it would contain %20 instead of a space).